app/src/main/java/org/launchcode/g7/bookcollections/NewBookFragment.java

```diff
@@ -26,6 +44,10 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
             public void onClick(View v) {
                 ISBN = editText.getText().toString();
                 //TODO do something with ISBN before removing fragment
+
+                // Send the newly created book back to the listener
+                onAddBookClickListener.onAddBookClick(new Book(ISBN));
```

Comment:
This is where the "magic happens." This is where the event is created that sends the new Book object back to its listeners. It's listener being MainActivity.